app/src/main/java/com/github/onlynotesswent/ui/common/Popups.kt

```diff
@@ -246,13 +299,18 @@ fun FileSystemPopup(onDismiss: () -> Unit, folderViewModel: FolderViewModel) {
                     }
                   }
                 }
-
-            Box(modifier = Modifier.fillMaxWidth().padding(8.dp)) {
-              ElevatedButton(
-                  onClick = { selectedFolder = null }, modifier = Modifier.fillMaxWidth()) {
-                    Text(stringResource(R.string.file_system_back_to_root))
-                  }
-            }
+            Box(
```

Comment:
As a suggestion, you could make this button a separate composable to make the `FileSystemPopUp` more modular

Reply:
I do not think it is a good idea as the code is quite small, not reused anywhere and integrated in this popup so I think it is a bad idea to modularize it.

Reply:
Even though it is not reused, it would improve the readability of the function